hackathon_site/hardware/test_api.py

```diff
@@ -1061,6 +1061,43 @@ def test_submitting_order_after_end_date(self):
         expected_response = {"non_field_errors": ["Hardware sign out period is over"]}
         self.assertEqual(response.json(), expected_response)
 
+    @override_settings(
+        HARDWARE_SIGN_OUT_START_DATE=datetime.now(settings.TZ_INFO)
+        + relativedelta(days=1)
+    )
+    def test_submitting_order_as_test_user_before_start_date_success(self):
```

Comment:
do we need a test to test if the user doesn't belong to the test user group?

Reply:
those tests already exists, they are functions `test_submitting_order_after_end_date` `test_submitting_order_before_start_date`